core/consensus/consensus_event.hpp

```diff
@@ -4,25 +4,41 @@
 #include <string>
 #include <functional>
 #include <memory>
+#include <type_traits>
 
 #include "../model/transactions/abstract_transaction.hpp"
 
 namespace consensus_event {
 
 struct ConsensusEvent {
+
     std::unique_ptr<abstract_transaction::AbstractTransaction> tx;
     std::vector<std::string> signatures;
     std::string merkleRoot;
+
     unsigned long long order = 0;
     std::vector<std::string> merkleRootSignatures;
 
-    ConsensusEvent(std::unique_ptr<abstract_transaction::AbstractTransaction> tx) {
-        
+    ConsensusEvent(std::unique_ptr<abstract_transaction::AbstractTransaction> atx):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Struct 'ConsensusEvent' has a constructor with 1 argument that is not explicit.](https://www.codacy.com/app/soramitsu/iroha/file/4274201852/issues/source?bid=3712501&fileBranchId=3837061#l22)